src/containers/Register.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import { Container } from 'react-bootstrap';
+
+class Register extends Component {
```

Comment:
There's already a registration page. You can delete this component.

Reply:
Strange, this file was included in the commit not on purpose.